constructor/winexe.py

```diff
@@ -86,6 +86,23 @@ def insert_tempfiles_commands(paths: os.PathLike) -> List[str]:
         lines.append(f"File {path}")
     return lines
 
+def setup_extra_env_variables(info) -> List[str]:
+    """Helper function to insert extra environment variables into nsis template.
+
+    Args:
+        info: Dictionary of information parsed from construct.yaml
+
+    Returns:
+        List[str]: Commands to be inserted into nsi template
+    """
+    lines = []
+    for variable in info.get('extra_env_variables',[]):
+        split = variable.split('=')
+        if len(split)==2:
+            (name, val) = split
```

Comment:
```suggestion
    for name, value in info.get('extra_env_variables', {}).items():
```

Reply:
I was planning to think about validation for the case of a missing `=` but that is moot now we are using yaml dicts instead. Happy to apply this suggestion.